### DISCUSSION TITLE
Nextcloud versions Compatibility

### DISCUSSION THREAD

**Discussion by TheMiNuS** [baimard/gestion#372] [General]
Hello,

I'm a nextcloud and Gestion user for a while and first I want to thanks all contributors as it's super usefull for small structures. 
What brings me here is regarding the compatibility with Nextcloud 28. I want to know if someone already tested the compatibility with NC28 ?

Regards,
MiNuS

**Comment by baimard** [comment]
Hello,

I'm fighting with webpack for a new release. I will test in few time.

regards,
Benjamin

**Reply by TheMiNuS** [reply]
Ok thanks for the update and no pressure.

**Reply by baimard** [reply]
thank you :), I really want to add the capability to share compagny datas between users.

**Reply by TheMiNuS** [reply]
That would be a great feature.

**Comment by baimard** [comment]
Ok soo hi check documentation for nextcloud 28 and there are big changes for applications...

Now, gestion is not compatible but I'm releasing a new version in 2.5.0 with also big changes for gestion and that will be compatible.

!

**Reply by TheMiNuS** [reply]
ok cool

**Reply by ctc** [reply]
When to expect this update?

**Reply by baimard** [reply]
it's up.

**Reply by ctc** [reply]
How? Installed dev-2023 - still can't change customer at offers.
And theres a typo:
gestion/lib/Controller/PageController.php Line 263
private function getCompaniesList(){
   $CompaniesList = $this->myDb->getCompaniesList($this->$myID); <-- myID and and $myID

**Reply by baimard** [reply]
The new release is in master, and work with NC 28.

Thank you for typo

**Reply by ctc** [reply]
master does not work for me with NC 28.0.2

**Reply by baimard** [reply]
Could you tell me more about your configuration ?

**Reply by ctc** [reply]
debian 11.5 / nginx 1.18.0 / php-fpm 8.1.27 / mariadb 10.5.15

**Reply by TheMiNuS** [reply]
It's working fine for me with 28.0.6
I hope we will have a release for NC 29.0.1 soon.
